### PR TITLE
BUGFIX: Spontaneous magic and magical effects are affected now by act…

### PR DESCRIPTION
**Pull request by LudoBermejoES**
BUGFIX: Spontaneous magic and magical effects are affected now by active effects related with spellcasting

### CODE REVIEW

File: templates/actor/parts/actor-arts.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
               class="flexrow fontSize10 rollable"
               name="data.arts.techniques.{{key}}.label"
               data-roll="spont"
+              data-bonus-active-effects="{{ ../data.bonuses.arts.spellCasting }}"7
```
Comment (Xzotl42) [inline templates/actor/parts/actor-arts.html:25]: Pretty sure the "7" at the end of the line 25 is a typo ;)

**Review by Xzotl42 — CHANGES_REQUESTED** [review]
Looks good except one typo (see comment)

File: templates/actor/parts/actor-arts.html (hunk 1 above)
Comment (LudoBermejoES) [reply templates/actor/parts/actor-arts.html]: Done!

Comment (LudoBermejoES) [reply templates/actor/parts/actor-arts.html]: Regression fixed, too

**Pull request merged**